tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -662,3 +662,197 @@ jobs:
     ArtifactSuffix: 'GPU'
     NugetPackageName : 'Microsoft.ML.OnnxRuntime.Gpu'
 
+- template: nuget/templates/win-ci-2019.yml
```

Comment:
This part is copied from tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml without change.